environment.yml

```diff
@@ -1,16 +1,17 @@
 name: minian
 channels:
   - conda-forge
+  - simpleitk
```

Comment:
@phildong can you please also update the requirements-base.txt in the requirements folder?
this is needed to install the MiniAn package through pip